exoctk/contam_visibility/sossContamFig.py

```diff
@@ -6,14 +6,231 @@
 from bokeh.layouts import gridplot
 from bokeh.plotting import figure
 from bokeh.models import Range1d, LinearColorMapper, Label
+from bokeh.models.widgets import Panel, Tabs
 from bokeh.palettes import inferno
 import numpy as np
 
 from . import visibilityPA as vpa
 
+TRACES_PATH = os.path.join(os.environ.get('EXOCTK_DATA'),  'exoctk_contam', 'traces')
 
-def contam(cube, targetName='noName', paRange=[0, 360], badPA=[], tmpDir="",
-           fig='', to_html=True):
+def tolam(x):

```

Comment:
Just wondering about where this function came from. Is this new or was it commented out in a previous commit?

Reply:
@mfixstsci this was a new function I made. It's original purpose was to calculate the values in lines 24-27 of this code, so it's no longer used in the code. I can remove this before merging.